pages/common/starship.md

```diff
@@ -0,0 +1,36 @@
+# starship
+
+> The minimal, blazing-fast, and infinitely customizable prompt for any shell.
+> More information: <https://starship.rs/>.
+
+- Print the shell function used to execute starship:
+
+`starship init {{bash|fish|zsh}`
```

Comment:
Is there a reason why you didn't include all supported shells?
```suggestion
`starship init {{bash|zsh|fish|powershell|ion|elvish|tcsh}}`
```
(and is there a reason why you ended the token syntax with only one `}`? 😄)